.eslintrc.js

```diff
@@ -2,23 +2,21 @@ module.exports = {
   env: {
     browser: true,
     commonjs: true,
-    es6: true,
+    es6: true
   },
-  extends: [
-    'airbnb-base',
-  ],
+  extends: ['airbnb-base'],
   globals: {
     Atomics: 'readonly',
-    SharedArrayBuffer: 'readonly',
+    SharedArrayBuffer: 'readonly'
   },
   parserOptions: {
-    ecmaVersion: 2018,
+    ecmaVersion: 2018
   },
   rules: {
-    'vars-on-top': "off",
+    'vars-on-top': 'off',
     'no-case-declarations': 'off',
-    'radix': 'off',
+    radix: 'off',
```

Comment:
why? This property becomes inconsistent with the others...

Reply:
I'll save this without formatting to keep this from switching.